service/geopmdpy/runtime.py

```diff
@@ -238,13 +247,16 @@ def run(self):
             pid = subprocess.Popen(self._argv)
             for loop_idx in TimedLoop(self._update_period, self._num_update):
                 if pid.poll() is not None:
+                    # Nice process has died error, and RC
```

Comment:
Totally, we should be tracking the return code of the subprocess and add an accessor method to the Controller object so it can be returned from main (should update the "test" accordingly).